### DISCUSSION TITLE
Windows 11 Change Language Switcher

### DISCUSSION THREAD

**Discussion by ilyfairy** [valinet/ExplorerPatcher#3209] [Q&A]
How to change the Language Switcher when the Taskbar style is Windows 11.
The Language Switcher option will only appear if the Taskbar style is Windows 10.
Is there a way to change the Language Switcher independently?

**Comment by pyrates999** [comment]
Does this help? https://www.elevenforum.com/t/show-language-bar-on-desktop-or-taskbar-in-windows-11.4107/

**Reply by ilyfairy** [reply]
I want to switch this
![image](https://github.com/valinet/ExplorerPatcher/assets/52264192/c5e25b3f-efae-45f0-802a-b3832bd519ee)

Win11 is too slow, select language through Ctrl+Space
![image](https://github.com/valinet/ExplorerPatcher/assets/52264192/de17d6a3-b022-4332-a7a0-dd256e1ea42e)

**Reply by pbaykalov** [reply]
Don't know how it was half a year ago but it is available already.
![image](https://github.com/user-attachments/assets/fd7d4740-ad9f-4858-aac8-0684914bc919)

Also, delay of W11 switcher is variable and it's large by default. Accidentially found out how it is controlled:
https://github.com/valinet/ExplorerPatcher/discussions/2126

**Comment by pyrates999** [comment]
Try this: https://answers.microsoft.com/en-us/windows/forum/all/how-to-easily-switch-from-keyboard-languages-on/9e9aaa8e-4311-4fee-b759-88f3fbed5910
